### DISCUSSION TITLE
Unblock repl.co with bookmarklet

### DISCUSSION THREAD

**Discussion by Theblackmamba824** [3kh0/ext-remover#218] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
can you make a bookmarklet that unblocks repl.co

**Comment by ghost** [comment]
the bookmarklet, when fixed, will unblock every site

**Comment by RuralAnemone** [comment, marked as answer]
get your own domain from [freenom](https://my.freenom.com) and connect it to the repl

**Reply by Theblackmamba824** [reply]
blocked

**Reply by RuralAnemone** [reply]
_skill issue_
or use your home computer if you have one
or your phone if you have one

**Comment by ILikeBypassing** [comment]
use a proxy

**Reply by RuralAnemone** [reply]
or a vpn

**Reply by ILikeBypassing** [reply]
most people have an extension so you can't use a vpn

proxy is still the better choice since you can deploy your own for free
